PROCESS.md

```diff
@@ -0,0 +1,163 @@
+# JSON Schema Specification Development Process
+This document formally defines the process used to evolve the JSON Schema
+specification as of the first stable release in 2023. It applies to the core
+specification and the specifications for any dialects and vocabularies
+maintained by the JSON Schema Org. It does not necessarily apply to everything
+the JSON Schema Org maintains. For example, media type registration follows the
```

Comment:
Should we also say that it doesn't apply to vocabularies (like mine) or other extensions (like JSON Schema form generation) defined outside of the org?

Reply:
I thought I had covered that by explicitly stating what this process applies to (specs maintained by the JSON Schema Org), but technically it doesn't say that's the only thing it applies to. I'll edit to make that less ambiguous.